src/App.jsx

```diff
@@ -3,24 +3,14 @@ import './App.css'
 import Layout from './views/Layout/Layout'
 import Home from './views/Home/Home'
 import { fetchUser } from './services/user'
+import { useUser } from './context/UserContext'
```

Comment:
Try to remember to remove unsued imports when you've refactored. This ensures that you're always submitting clean and maintainable code. :)